pages/myProfile/userBio/userBio.tsx

```diff
@@ -0,0 +1,58 @@
+import { UserCounter } from '@/pages/myProfile/userCounter/userCounter';
+import { Button } from '@/shared/ui';
+import { AvatarRadix } from '@/shared/ui/avatar';
+import { Typography } from '@/shared/ui/typography';
+
+import s from './userBio.module.scss';
+
+export const UserBio = () => {
+  const profile = {
+    familyName: 'inst',
+    firstName: 'ana',
+    followers: 2358,
+    following: 2218,
+    photos: [
+      {
+        authorId: '32asdf67-283b-16d7-a546-4266as4400fe',
+        createdAt: '2023-10-26T12:55:21.448Z',
+        cropProps: '{"pos":{"x":1,"y":0.5},"scale":1}',
+        id: '32asdf67-283b-16d7-a546-4266as4400fe',
+        title: 'vacations-2004.jpg',
+        updatedAt: '2023-10-26T12:55:21.448Z',
+        url: 'http://photo.com',
+      },
+    ],
+    publication: 2764,
+    status:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
+      'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
+      'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ' +
+      'ut aliquip ex ea commodo consequat.',
+    userName: 'Instagirl',
+  };
+
+  return (
+    <div className={s.userBioRoot}>
+      <div className={s.userBioAvatar}>
+        <AvatarRadix urlAdress={profile.photos[0].url} />
```

Comment:
Missing `d` in `urlAdress`. Please, install spell checking plugin for your IDE. In Webstorm it is already installed. In VSCode you can try [CSpellChecker]()